.rubocop.yml

```diff
@@ -0,0 +1,21 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
let's discuss in https://github.com/reallyenglish/ansible-role-example/issues/60.